storage/lmdb/src/remap.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Support for memory map reallocation
+
+use logging::log;
+use utils::tap_error_log::LogError;
+
+/// Represents LMDB memory map size
+#[derive(Eq, PartialEq, PartialOrd, Ord, Clone, Copy, Debug)]
+pub struct MemSize(u64);
+
+impl MemSize {
+    /// Specify in the number of bytes
+    pub const fn from_bytes(bytes: u64) -> Self {
+        Self(bytes)
+    }
+
+    /// Specify in the number of kilobytes
+    pub const fn from_kilobytes(kilobytes: u64) -> Self {
+        Self::from_bytes(1024 * kilobytes)
+    }
+
+    /// Specify in the number of megabytes
+    pub const fn from_megabytes(megabytes: u64) -> Self {
+        Self::from_kilobytes(1024 * megabytes)
+    }
+
+    /// Convert to raw byte count
+    pub fn as_bytes(self) -> usize {
+        self.0.try_into().expect("Ran out of address space")
+    }
+
+    pub fn div_ceil(self, rhs: Self) -> u64 {
+        self.0 / rhs.0 + (self.0 % rhs.0 > 0) as u64
+    }
+}
+
+impl std::fmt::Display for MemSize {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}B", self.0)
```

Comment:
The crate `byte_unit` can do this nicely for you.

Reply:
Do you think it's worth pulling in given it's just used for logging?

Reply:
Meh... I don't know... probably not.